spec/helpers/pageflow/admin/users_helper_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+module Pageflow
+  module Admin
+    describe UsersHelper do
+      describe '#delete_own_user_section' do
+        it 'renders may_delete if authorize_user_deletion gives true' do
+          Pageflow.config.authorize_user_deletion = lambda { |_user| true }
+
+          expect(helper).to receive(:current_user).and_return(create(:user))
+          expect(helper).to receive(:delete_me_admin_users_path).and_return('#')
+
+          section = helper.delete_own_user_section
+
+          expect(section).to have_selector('.may_delete')
+          expect(section).not_to have_selector('.cannot_delete')
+        end
+
+        it 'renders cannot_delete if authorize_user_deletion gives any string' do
+          Pageflow.config.authorize_user_deletion = lambda { |_user| 'true' }
```

Comment:
This is confusing. The quotes are hard to spot. Return a string like "Not allowed" and also query for it inside the resulting HTML.